### ISSUE TITLE
moving nodes of a polygon has a problem with the index

### ISSUE THREAD

**Issue by geohacker** [developmentseed/observe#258]
I think when you select node 1, node 2 is moving or something of that sort.

**Comment by sethvincent** [comment]
I haven't quite figured this out yet, but it does seem like this issue only comes up when trying to move the first/last node.

Image that shows the last node moving but not the first:
![Screen Shot 2020-06-02 at 4 41 21 PM](https://user-images.githubusercontent.com/164214/83580349-b68cab80-a4f0-11ea-99a4-4069add07624.png)


The coordinates array at this point in the newWay looks like this:

```
[
  [-122.3518706, 47.659928]
  [-122.35172222257684, 47.65990543303783]
  [-122.35182682874253, 47.660045440372514]
  [-122.35191355347801, 47.659904228762024]
]
```

I think we need to test if we're trying to move the first/last node in a polygon and then move them both?

I've also noticed that this only matters in the editing styles, the correct polygon shape gets submitted in the changeset.

**Comment by sethvincent** [comment]
alright i tried out the idea i outlined in the previous comment: https://github.com/developmentseed/observe/pull/287/files#diff-a15753485c986d85ba0326dec6ba0167R19-R25

it feels like messy code but it resolves the thing i was seeing.

**Comment by geohacker** [comment]
This is fixed in #287
